### PR TITLE
Review of Installation doc

### PR DESCRIPTION
**Pull request by Preeticp**
## What is the purpose of this change? What does it change?
This PR reviews and modifies the installation doc and converts this doc to asiccdoc format.

### CODE REVIEW

**Comment by Preeticp** [comment]
@surajnarwade @cdrage please review. In my next commit I will convert this to Asciidoc format

File: docs/installation.md [hunk 1]
```diff
@@ -1,151 +1,122 @@
...
-
-```sh
-# CentOS / RHEL
-yum install odo
+2. Install Odo:
 
-# Fedora
-dnf install odo
- ```
+    * For CentOS or RHEL use `yum install odo`.
```
Comment (surajnarwade) [inline docs/installation.md]: command on next line will look good

File: docs/installation.md [hunk 2]
```diff
@@ -1,151 +1,122 @@
...
+    * For Fedora use `dnf install odo`.
```
Comment (surajnarwade) [inline docs/installation.md]: same here

File: docs/installation.adoc [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+[bintraybintray-odo-odo-rpm-releases]
+name=bintray-odo-odo-rpm-releases
+baseurl=https://dl.bintray.com/odo/odo-rpm-releases
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+....
+. Install Odo:
+* For CentOS or RHEL use `yum install odo`.
+* For Fedora use `dnf install odo`.
```
Comment (surajnarwade) [inline docs/installation.adoc]: command on next line

Comment (Preeticp) [reply docs/installation.adoc]: Fixed in the .adoc. we will be deleting the .md file, so not changing there.

**Comment by Preeticp** [comment]
@surajnarwade @cdrage if this is good I will delete the .md file.

**Comment by cdrage** [comment]
@Preeticp `.md` file must be kept until the docs script is completed. Deleting `readme.md` is fine, but nothing in the `docs` folder yet..

**Comment by cdrage** [comment]
This doc LGTM!

/lgtm

**Comment by cdrage** [comment]
@surajnarwade do the approve command when you're ready! :)

**Pull request merged**